scripts/validate_schemas.py

```diff
@@ -0,0 +1,137 @@
+import glob
+import json
+import logging
+import os
+import re
+import subprocess
+import sys
+import time
+from concurrent.futures import ThreadPoolExecutor, as_completed
+
+logging.basicConfig(
+    level=logging.INFO, format="%(asctime)s - %(levelname)s - %(message)s"
+)
+
+
+def check_connection():
+    for connection_attempts in range(4, 0, -1):
+        response = subprocess.run(
+            [
+                "curl",
+                "-so",
+                "/dev/null",
+                "-w",
+                "%{http_code}",
+                "http://localhost:5002/status",
+            ],
+            capture_output=True,
+            text=True,
+            check=False,
+        ).stdout.strip()
+
+        if response == "200":
+            return
+
+        logging.error("\033[31m---Error: Schema Validator Not Reachable---\033[0m")
+        logging.error(  # pylint: disable=logging-too-many-args
+            "\033[31mHTTP Status: %s\033[0m", response
+        )
+
+        if connection_attempts == 1:
+            logging.info("Exiting...\n")
+            sys.exit(1)
+
+        logging.info("Retrying...\n")
+        time.sleep(5)
+
+
+def get_schemas() -> list[str]:
+    arguments = sys.argv
+    # argv[0] is the script name, so len(argv) == 1 means no arguments passed
+    if len(arguments) == 1 or arguments[1] == "--local":
+        file_path = "./schemas"
+
+    else:
+        file_path = arguments[1]
```

Comment:
Could do this inline?
```suggestion
    # argv[0] is the script name, so len(argv) == 1 means no arguments passed
    file_path = "./schemas" if len(sys.argv) == 1 or sys.argv[1] == "--local" else sys.argv[1]
```